packages/types/src/index.ts

```diff
@@ -300,6 +300,7 @@ export interface InputBlock extends Block {
   hint?: PlainTextElement;
   optional?: boolean;
   element: Select | MultiSelect | Datepicker | PlainTextInput | RadioButtons | Checkboxes;
+  dispatch_actions?: boolean;
```

Comment:
The name in the document is incorrect (I've already reported it internally). You can verify `dispatch_action` works while `dispatch_actions` doesn't [here](https://app.slack.com/block-kit-builder#%7B%22type%22:%22modal%22,%22title%22:%7B%22type%22:%22plain_text%22,%22text%22:%22My%20App%22,%22emoji%22:true%7D,%22submit%22:%7B%22type%22:%22plain_text%22,%22text%22:%22Submit%22,%22emoji%22:true%7D,%22close%22:%7B%22type%22:%22plain_text%22,%22text%22:%22Cancel%22,%22emoji%22:true%7D,%22blocks%22:%5B%7B%22dispatch_action%22:true,%22type%22:%22input%22,%22element%22:%7B%22type%22:%22plain_text_input%22,%22action_id%22:%22plain_text_input-action%22%7D,%22label%22:%7B%22type%22:%22plain_text%22,%22text%22:%22Label%22,%22emoji%22:true%7D%7D%5D%7D).

```suggestion
  dispatch_action?: boolean;
```